pkg/kubelet/nvidiagpu/nvidia_gpu_manager.go

```diff
@@ -0,0 +1,190 @@
+/*
```

Comment:
Any chance we can change the import path to something like ``pkg/kublet/gpu/nvidia/PACKAGE`` ? I have a followup GPU PR that is using this layout to add some more nvidia packages.